src/main/java/core/mate/academy/service/BulldozerProducer.java

```diff
@@ -0,0 +1,21 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Bulldozer;
+import java.util.ArrayList;
+import java.util.List;
+
+public class BulldozerProducer implements MachineProducer<Bulldozer> {
```

Comment:
Since `BulldozerProducer` and all created by you producers are actual implementations, it's good to add `Impl` at the end. For example `BulldozerProducerImpl`.